README.md

```diff
@@ -1,48 +1,66 @@
 # HackBaltimore
 
-Get Ruby on Rails running on your computer
+The HackBaltimore website is statically generated using jekyll and GitHub pages.
 
-1. bundle exec install
+This README assumes that the reader has previous knowledge and experience using Ruby on Rails, jekyll, GitHub pages, JSON, and related web dev technologies.
+
+To edit the website locally you must first:
+1. Get Ruby on Rails running on your computer
```

Comment:
We don't need rails for this. Just the later steps.

Reply:
so just ruby is needed?

Reply:
Yeah, good point. Just ruby, bundler, then bundle install should get it going.